### DISCUSSION TITLE
Adding custom blocklist (with absolute path) throws error "does not match safe patterns"

### DISCUSSION THREAD

**Discussion by njscorpion** [AdguardTeam/AdGuardHome#7325] [Q&A] [answered]
i have setup my AdGuard (x64) on raspberry pi 5 on raspbian os. all was working fine till the latest update (well it still is) the only problem i am facing after updating to v0.107.53 is when trying to add a filter (Filters > DNS blocklists > Add blocklist > Add a custom list) this is absolute path that i am trying to add ex: /home/[user]/pvtlist/MyBlockList.txt , this used to work earlier but now i keep getting the error "does not match safe patterns" 

![image](https://github.com/user-attachments/assets/e7532520-ca1c-419c-bca1-34e942c06a3a)

Please Assist, Thanks

**Comment by ainar-g** [comment, marked as answer]
See [v0.107.53 release notes](https://github.com/AdguardTeam/AdGuardHome/releases/tag/v0.107.53).  You can either add `/home/…/pvtlist/*` to `filtering.safe_fs_patterns` or put your list files into the default safe directory, which is `data/userfilters` in AdGuard Home's working dir.

**Reply by njscorpion** [reply]
Thanks a lot

**Comment by graysky2** [comment]
found this on a search, to be clear, at least based on my `/etc/adguardhome.yaml`, the directory is 
```
grep -A 1 safe_fs_patterns: /etc/adguardhome.yaml
  safe_fs_patterns:
    - /var/lib/adguardhome/userfilters/*
```

So no data in the mix.

**Reply by Vasil-Dimitrov** [reply]
The lack of proper documentation on this is diabolical. Your comment was the solution. 

To make it accept my custom list on windows I had to place it under ./AdGuardHome/userfilters/
Only then it accepted my absolute path and creates a copy of my custom list under /AdGuardHome/data/filters/
